.gitignore

```diff
@@ -19,3 +19,4 @@ target
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+.DS_Store
```

Comment:
yeetni ty komentáře nad tim, .idea se tě skoro určitě netýká (nebo ty to píšeš v něčem Jetbrains?) a určitě přidej target/